libs/pxweb2-api-client/src/core/OpenAPI.ts

```diff
@@ -20,9 +20,9 @@ export type OpenAPIConfig = {
 };
 
 export const OpenAPI: OpenAPIConfig = {
-    BASE: 'https://api.scb.se/OV0104/v2beta/api/v2',
+    //BASE: 'https://api.scb.se/OV0104/v2beta/api/v2',
     //BASE: 'https://pxapi2-master-px.scb.se/api/v2',
-    //BASE: 'https://data.qa.ssb.no/pxapi2-beta/api/v2',    
+    BASE: 'https://data.qa.ssb.no/pxapi2-beta/api/v2',    
```

Comment:
Should the URL that is uncommented here be the first one? The one to api.scb.se?

Reply:
Or hsould I remove all of them since the value is acctually pick set from the config.js file to prevent confusion?

Reply:
That might be better yes

Reply:
Unforotunatly you have to set BASE to something so I changed to to statistics swedens beta site and I also added a comment that it is overriden in config.js